server/models/Conversation.js

```diff
@@ -0,0 +1,25 @@
+const mongoose = require('mongoose');
+
+const { Schema } = mongoose;
+
+const ConversationSchema = new Schema({
+    // group information
+    group: {
+        type: Schema.Types.ObjectId,
+        required: true,
+        ref: 'Group',
+    },
+    // individual message information
+    messages: [
+        {
+            user: { type: Schema.Types.ObjectId },
+            dateCreated: {
+                type: Date,
+                default: Date.now,
```

Comment:
I think this default value is a function. Might need to change this to Date.now().

Reply:
I guess mongoose executes it as a Date.now(). Because I've used it like that before, and I got date and time in the database.

Reply:
Oh cool never mind then @jonquag !